plugins/foreman_discovery/nightly/index.md

```diff
@@ -367,6 +366,14 @@ change the default entry from localboot entry to Discovery:
 
     ONTIMEOUT discovery
 
+Similar change is needed in *PXEGrub2 global default* template:
+
+    default=discovery
+
+*PXEGrub1 global default* must be changed in the same way, but this is a legacy
+bootloader not in wide use anymore. In that case provide number as the Grub1
```

Comment:
what does `number` refer to? "the number of the menu entry"?

Reply:
Yes, grub1 only supports indexed selection.